src/CovidSim.cpp

```diff
@@ -5004,12 +4954,12 @@ void RecordSample(double t, int n)
 				{
 					s = ((double)trigAlert)/((double)P.AlertTriggerAfterIntervThreshold);
 					thr = 1.1 / sqrt((double)P.AlertTriggerAfterIntervThreshold);
-					if (thr < 0.05) thr = 0.05;
+					if (thr < 0.05) thr = 0.04;
```

Comment:
Is this change correct or incomplete - it looks odd?  Did you mean to change the 0.05 to 0.04 - or did you miss the condition check earlier and fail to change that too?